.github/workflows/deploy.yml

```diff
@@ -69,3 +76,23 @@ jobs:
           app-name: ${{ vars.AZURE_WEBAPP_NAME }}
           images: ghcr.io/${{ github.repository }}:${{ github.sha }}
           publish-profile: ${{ secrets.AZURE_WEBAPP_PUBLISH_PROFILE }}
+
+  release:
+    needs: deploy
+    if: ${{ (github.ref_type != 'tag' && github.ref_name || contains(github.ref, '-rc') && 'test' || 'prod') == "prod" }}
```

Comment:
I get that you're just re-using the same expression here, which is fine... but really all this check cares about is if it's a production tag, we don't need the env name or anything else here.

Maybe this could be simplified to something like `${{ github.ref_type == "tag" && !contains(github.ref, "-rc") }}`?

Reply:
That's true, this expression works better! 👍